client/src/partials/reports/taxes_payment/taxes_payment.html

```diff
@@ -20,8 +20,7 @@
                     <span class="input-group-addon">
                       <span class="glyphicon glyphicon-folder-open"></span>
                     </span>  
-                    <select class="form-bhima" required ng-click="selecTaxes()" ng-model="session.period_id">
-                      <option ng-repeat="getPeriod in getPeriods.data" ng-value="getPeriod.id">{{getPeriod.label}} :: {{getPeriod.dateFrom | date }} - {{getPeriod.dateTo | date }}</option>
+                    <select class="form-bhima" required ng-value="getPeriod.id" ng-click="selecTaxes()" ng-options="getPeriod.id as format(getPeriod) for getPeriod in getPeriods.data" ng-model="session.period_id">
```

Comment:
I don't think you need ng-value here.  If you have an ng-model, then it should automatically bind your value to the ng-model.

Also, use ng-change="selectTaxes()", rather than "ng-click=selectTaxes()".  Since you are using a _select_ it is more meaningful than _click_ which is for buttons.